Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -3,29 +3,25 @@
 using System;
 
 [Serializable]
-public class LightSettings
+public class TimeRenderSettings
 {
-	public Color color;
-	public float intensity;
-}
+	public Color color = Color.white;
 
-[Serializable]
-public class FogSettings
-{
-	public Color color;
-	public float density;
+	public Color lightColor = Color.white;
+	public float lightIntensity = 0.5f;
+
+	public Color fogColor = Color.white;
+	public float fogDensity = 0.05f;
```

Comment:
These were chosen on purpose. I think 0.5f is the default light setting in Unity, 0.05f is an obvious fog and then White is an obvious color. Do you think I should use something that looks like a real setting as the default or something obscene so we actually set these?